src/main/java/com/sprint/monew/domain/interest/CursorPageResponseInterestDto.java

```diff
@@ -0,0 +1,14 @@
+package com.sprint.monew.domain.interest;
+
+import java.time.Instant;
+import java.util.List;
+
+public class CursorPageResponseInterestDto {
+
+  private List<InterestDto> content;
+  private String nextCursor;
+  private Instant nextAfter;
+  private int size;
+  private int totalElements;
+  private boolean hasNext;
+}
```

Comment:
각각의 DTO에 대한 CursorPageResponse를 만들기보다 제네릭으로 선언해서 사용하는게 어떤가요? 여기도 record를 사용하는게 좋을거같네요.
```suggestion
public record CursorPageResponse<T>(
    List<T> content,
    Object nextCursor,
    int size,
    boolean hasNext,
    Long totalElements
) {
}
```